components/sup-protocol/build.rs

```diff
@@ -22,6 +22,8 @@ use prost_types::{DescriptorProto,
                   FileDescriptorProto,
                   FileDescriptorSet};
 
+const GENERATED_OUT_DIR: &'static str = "src/generated";
```

Comment:
```suggestion
const GENERATED_OUT_DIR: &str = "src/generated";
```
`const` lifetimes can be inferred now